include/api.php

```diff
@@ -41,24 +41,45 @@ function pll_the_languages( $args = array() ) {
 
 /**
  * Returns the current language on frontend.
- * Returns the language set in admin language filter on backend ( false if set to all languages ).
+ * Returns the language set in admin language filter on backend (false if set to all languages).
  *
  * @api
  * @since 0.8.1
  *
- * @param string $field Optional, the language field to return ( @see PLL_Language ), defaults to 'slug'. Pass OBJECT constant to get the language object.
- * @return string|PLL_Language|false The requested field or object for the current language, false if field isn't set or if current language doesn't exist yet.
+ * @param string $field Optional, the language field to return (@see PLL_Language), defaults to 'slug'. Pass OBJECT constant to get the language object.
+ * @return string|int|bool|string[]|PLL_Language The requested field or object for the current language, false if the field isn't set or if current language doesn't exist yet.
+ *
+ * @phpstan-param \OBJECT|'name'|'slug'|'locale'|'w3c'|'flag_code'|'host'|'flag_url'|'flag'|'home_url'|'search_url'|'facebook'|'custom_flag_url'|'custom_flag'|'mo_id'|'page_on_front'|'page_for_posts'|'term_id'|'term_taxonomy_id'|'count'|'language:term_id'|'language:term_taxonomy_id'|'language:count'|'term_language:term_id'|'term_language:term_taxonomy_id'|'term_language:count'|'term_group'|'is_rtl'|'active'|'fallbacks'|'is_default' $field
```

Comment:
Can `@phpstan-param` be imported here?

Reply:
Nop, it can be imported into classes only.
Anyway, I simplified things in [9dbdf67](https://github.com/polylang/polylang/pull/1227/commits/9dbdf67c2683a050a16451823818b6f91a2ed671).